site/docs/src/json/applications/response.json

```diff
@@ -83,11 +86,15 @@
       }
     ],
     "samlv2Configuration" : {
-      "callbackURL": "https://www.example.com/samlv2/acs",
+      "authorizedRedirectURLs": [
+        "https://www.example.com/samlv2/acs"
+      ],
```

Comment:
This one is tricky, for backwards compatibility, we are actually keeping this field as well. 

So you'll want to keep this in the example JSON response:

```
"callbackURL": "https://www.example.com/samlv2/acs",
```

It is ok to leave out of the request example because- while it will work, it is not the expected request body. 

See this code to see how we are doing this: 

```java
    /**
     * @return the configured callback URL or null if not defined.
     * @deprecated Prefer the use of {{@link #authorizedRedirectURLs}}
     */
    @Deprecated
    public URI getCallbackURL() {
      if (authorizedRedirectURLs.isEmpty()) {
        return null;
      }

      return authorizedRedirectURLs.get(0);
    }

    /**
     * @param callbackURL the callback URL
     * @deprecated Prefer the use of {{@link #authorizedRedirectURLs}}
     */
    @Deprecated
    public void setCallbackURL(URI callbackURL) {
      if (!authorizedRedirectURLs.contains(callbackURL)) {
        authorizedRedirectURLs.add(callbackURL);
      }
    }
```

Reply:
I did see that, but do we want to include `deprecated` fields in the JSON examples? Shouldn't we be steering people away from that usage?

Reply:
Never mind, reread the comment. I think we are good here.

Reply:
This is still missing, we need to add this to the JSON response. You can review test responses, or call the API manually and review the JSON response for context.